### PR TITLE
minimos: fixed chrome certificate issue.

### PR DESCRIPTION
**Pull request by sagarjminim**
SW-3193

### CODE REVIEW

**Comment by sagarjminim** [comment]
Acoording to the https://github.com/openwrt/openwrt/commit/3c4858eeb2bbb3107f87bb3be07d5c172c8e0ef  we need openssl-util package instead of px5g-standalone package to generate the ssl certificate. I have tested it and now the webpage is opening on the chrome with the proceed to unsafe option.

File: profiles/minim.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ packages:
   - kmod-nf-nathelper
   - kmod-nf-nathelper-extra
   - minim-web-admin
-  - px5g-standalone
+  #- px5g-standalone
```
Comment (bmastbergen) [inline profiles/minim.yml]: no need to leave the commented out px5g-standalone

Comment (sagarjminim) [reply profiles/minim.yml]: As per the discussion I am removing the px5g-standalone package and adding the openssl-util package.

Comment (bmastbergen) [reply profiles/minim.yml]: Right, but you don't need to have a line with a commented out px5g-standalone.  Remove it completely

Comment (sagarjminim) [reply profiles/minim.yml]: Ok got it.

Comment (sagarjminim) [reply profiles/minim.yml]: @bmastbergen Done.

**Review by bmastbergen — APPROVED** [review]

**Pull request merged**